src/windows/views/timeline_webview.py

```diff
@@ -2595,15 +2595,8 @@ def update_zoom(self, newValue):
 
     def keyPressEvent(self, event):
         """ Keypress callback for timeline """
-        key_value = event.key()
-        if (key_value == Qt.Key_Shift or key_value == Qt.Key_Control):
-
-            # Only pass a few keystrokes to the webview (CTRL and SHIFT)
-            return QWebView.keyPressEvent(self, event)
-
-        else:
-            # Ignore most keypresses
-            event.ignore()
+        #keyboards events are necessary for the time input field to work. No apparent conflict with the rest of the interface.
+        return QWebView.keyPressEvent(self, event)
```

Comment:
Not sure if a more specific filter is needed. Doesn't seems to conflict with any other shortcut in the editor.

Reply:
No, _conflicts_ per se don't seem to be a problem, agreed. One issue I have hit on in testing, though, is that the editable field makes it much easier to inadvertently end up focusing it — which will then interfere with several keyboard shortcuts, including <kbd>Space</kbd> to toggle Play/Pause.

To be fair, the Properties table has the same issue. After it's been focused for keyboard navigation, it can have a tendency to "re-grab" focus on subsequent inputs that you wouldn't expect to go to the table. But it's especially disconcerting here, because any click on the ruler will also focus the time field. So, you can't perform common actions like positioning the playhead with a ruler click, followed by <kbd>Space</kbd>to play from that point.

The timeline arrow-key navigation bindings, like <kbd>←</kbd>, <kbd>→</kbd>, <kbd>Ctrl</kbd>+<kbd>←</kbd>, <kbd>Ctrl</kbd>+<kbd>→</kbd>, etc. are also blocked by the focus grab, unless you first tab out of the field. (The last two <kbd>Ctrl</kbd> combos I mentioned will jump left or right to the next `:` in the time field when it's focused, interestingly.)

I'll comment more generally below, but I wanted to raise that issue here since it pertains to keypress events in particular. I don't think these issues are _unsolvable_, by any means, but... well, like I said, I'll leave an overview response in a separate PR comment.